### DISCUSSION TITLE
HA 2022.8.0 bluetooth

### DISCUSSION THREAD

**Discussion by ffrants** [TionAPI/HA-tion#101] [Q&A]
Кто-нибудь уже обновлялся на 2022.8? ) 
Работает интеграция? ) 
Они там завезли новую bluetooth интеграцию и есть подозрение что что-то может поломаться 
https://www.home-assistant.io/blog/2022/08/03/release-20228/#first-class-bluetooth-support

Или может вообще можно туда как-то наши тионы пропихнуть, чтобы они поддерживались искаропки? (Я плохо разбираюсь как это все работает, может фигню сказал) )

**Comment by IATkachenko** [comment]
Поломаться не должно: я работаю напрямую с BT стеком, а bleak сейчас нормально поддерживает асинхронность.

Если все будет нормально -- посмотрю в выходные что они там такого сделали клевого и может ли это быть нам полезно.

**Reply by ffrants** [reply]
Тогда обновляюсь 😁

**Reply by ffrants** [reply]
Не было печали, апдейтов накачали ))
Создал issue ))

**Reply by IATkachenko** [reply]
И не говорите, не дают спокойно жить, со своими обновлениями.

**Reply by ffrants** [reply]
Нашёл, кстати, интересный проект, сосед себе настроил, говорит что вроде стабильно работает 
https://github.com/dentra/esphome-tion
вот жду esp’хи из Китая чтобы попробовать

**Reply by IATkachenko** [reply]
Отличный проект :) Я тоже думал про хардварное подключение, чтобы исключить BTLE, но руки дошли только до BTLE версии, которой оказалось достаточно.
